linchpin/provision/roles/openshift/tasks/provision_resource_group.yml

```diff
@@ -36,6 +36,17 @@
   no_log: "{{ not debug_mode }}"
   when: res_grp['credentials'] is defined and res_grp['credentials']['filename'] is defined
 
```

Comment:
what do you think of adding debug task here to check the right workspace is being used:
- debug: msg="{{ lookup('env','workspace') }} is an workspace env variable"

Reply:
@skatlapa I tested it on my workspace , I can add that .

Reply:
+1